main/templates/main/index.html

```diff
@@ -9,118 +9,97 @@
+    <!-- Indicators 
+    <ol class="carousel-indicators">
+      {% for coverpic in coverpics %}
+        <li data-target="#carousel-main" data-slide-to="{{forloop.counter0}}" class="{% if forloop.counter == 1 %}active{% endif %}"></li>
+      {% endfor %}
+    </ol>-->
 
-<section id="section-stats">
-	<div class="container">
-		<h2>Enactus at the University of Waterloo</h2>
-		<div class="row">
-			<div class="col-md-4" id="stat-1">
-				<div class="fa-stack">
-				  <i class="fa fa-circle fa-stack-2x"></i>
-				  <i class="fa fa-users fa-stack-1x fa-inverse"></i>
-				</div>
+    <!-- Wrapper for slides -->
+    <div class="carousel-inner" role="listbox">
+      {% for coverpic in coverpics %}
+        <div class="item {% if forloop.counter == 1 %}active{% endif %} text-center">
+          <a href="{{coverpic.url}}" style="height: 100%; display: block;">
+            <img src="{{coverpic.image.url}}" alt="coverpic-{{forloop.counter}}">
+            <div class="carousel-caption">
+              <div class="carousel-caption-title">
+                <span class="carousel-title-rect">{{coverpic.title}}</span>
+              </div>
+              <div class="carousel-caption-subtitle">
+                <span class="carousel-subtitle-rect">{{coverpic.subtitle}}</span>
+              </div>
+            </div>
+          </a>
+        </div>
+      {% endfor %}
+    </div>
 
-				<div class="stat stat-number">35</div>
-				<div class="stat stat-type">Members</div>
-			</div>
+    <!-- Controls -->
+    <a class="left carousel-control" href="#carousel-main" role="button" data-slide="prev">
+      <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+      <span class="sr-only">Previous</span>
+    </a>
+    <a class="right carousel-control" href="#carousel-main" role="button" data-slide="next">
+      <span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+      <span class="sr-only">Next</span>
+    </a>
+  </div>
+</section>
 
-			<div class="col-md-4" id="stat-2">
-				<div class="fa-stack">
-				  <i class="fa fa-circle fa-stack-2x"></i>
-				  <i class="fa fa-building fa-stack-1x fa-inverse"></i>
-				</div>
+<section id="section-whatis-enactus">
+  <div class="container">
+    <h2>Who we are</h2>
+    <div class="section-text">
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer molestie nulla velit, a elementum nulla suscipit quis. Suspendisse lobortis nibh ex, lobortis pulvinar sem semper non. Nunc nec eros consequat, eleifend odio vel, molestie felis. Integer eget dolor nec sem bibendum condimentum. Ut pretium dapibus turpis id rhoncus. Nunc faucibus, risus vel tristique malesuada, diam nunc feugiat mauris, eu convallis velit ligula id justo. Mauris in feugiat eros. Vivamus accumsan eget ipsum id mollis. Aenean vitae rhoncus orci, id mattis justo. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.</p>
```

Comment:
Should we be pushing with filler text? I think we should have the correct text here, so that once we merge we can just deploy.